frontend/hub/approvals/hooks/useApprovalActions.tsx

```diff
@@ -70,6 +71,10 @@ export function useApprovalActions(callback?: (collections: CollectionVersionSea
         label: t('Reject'),
         onClick: (collection) => rejectCollections([collection]),
         isDanger: true,
+        isDisabled: (collection) =>
+          collection?.repository?.pulp_labels?.pipeline === 'rejected'
+            ? t`You do not have rights to this operation`
```

Comment:
I'd go with `true` if it can be used to disable without a message;
if not  .. this is not about rights, but more about ... this is already done.

So.... `` t`Collection is already rejected` `` or something like that?